metapool/tests/test_count.py

```diff
@@ -124,12 +124,14 @@ def test_parse_fastp_counts(self):
 
     def test_bcl2fastq_no_stats_file(self):
         bad_dir = os.path.join(os.path.abspath(self.run_dir), 'Trojecp_666')
-        with self.assertRaisesRegex(IOError, "Cannot find Stats.json '"
-                                             f"{bad_dir}/Stats/Stats.json' or "
-                                             "Demultiplex_Stats.csv '"
-                                             f"{bad_dir}/Reports/Demultiplex_"
-                                             "Stats.csv' for this run"):
-            bcl2fastq_counts(bad_dir, self.ss)
+
+        msg = (f"Cannot find Stats.json '{bad_dir}/Stats/Stats.json' or "
+               f"Demultiplex_Stats.csv '{bad_dir}/Reports/Demultiplex_Stats"
+               f".csv' or SeqCounts.csv '{bad_dir}/Reports/SeqCounts.csv' for"
+               " this run")
+
+        with self.assertRaisesRegex(IOError, msg):
+            raw_read_counts(bad_dir, self.ss)
 
     def test_bcl2fastq_counts_malformed_results(self):
```

Comment:
The modified test methods are no longer (directly) testing `bcl2fastq_counts`, so I strongly favor renaming them with the new tested function name (e.g., `test_raw_read_counts_malformed_results`

Reply:
ty good catch!